db/migrate/20160720085321_create_container_template.rb

```diff
@@ -0,0 +1,26 @@
+class CreateContainerTemplate < ActiveRecord::Migration[5.0]
```

Comment:
@zakiva I think it would be better to split this file to 2 files, one for each new table.